packages/back-end/src/services/experimentNotifications.ts

```diff
@@ -132,29 +127,20 @@ export const notifyAutoUpdate = ({
 export const notifyMultipleExposures = async ({
   context,
   experiment,
-  results,
-  snapshot,
+  currentStatus,
 }: {
   context: Context;
   experiment: ExperimentInterface;
-  results: ExperimentReportResultDimension;
-  snapshot: ExperimentSnapshotDocument;
+  currentStatus: RunningExperimentStatusData;
```

Comment:
question: So we only call this function for experiments that are in the `running` status?

Reply:
Hmm, we call it for all experiments, because `getRunningExperimentStatus` can be computed for stopped experiments as if they were running. Maybe it's a misnomer?

Reply:
Yeah -- to me this seems to be a subset of data that exists only for Experiments that are running. Should we rename it to not include the name of one of our statuses?